src/mailer.js

```diff
@@ -0,0 +1,39 @@
+const mailer = require('nodemailer');
+
+// create reusable transporter object using the default SMTP transport
+let transporter = mailer.createTransport({
+    service: 'gmail',
+    auth: {
+        user: 'fakehealthcareforyou@gmail.com',
+        pass: ''
+    }
+});
+
+const sendInvoice = (data) => {
+
+  let body = "Hello, and thank you for your recent visit to " +
+             "team null's hospital. We hope you are satisfied with your " +
+             "placebo treatment, we are sure you will be feeling better in " +
+             "null time. Now for the stuff we really care about: " +
+             "\n\nYour invoice total comes out to: $" + data.amount + ".\n\n" +
+             "You can pay this using your online invoice number: " +
+             data.id + " at [insert endpoint here]." +
```

Comment:
@dlowrey Alright, well I haven't setup the hosting for the webapp, so I can't give you a link yet. But we can make the change later